### ISSUE TITLE
Problem with the journal pop-up

### ISSUE THREAD

**Issue by Felipefpl** [Gibberlings3/EE_Fixpack#75]
We always see a pop-up in the top center of the screen showing when a new entry was added in the journal, in theory...

In theory because the pop-up lasts 2-3 seconds which makes it too hard to read anything nicely/carefully. It would be better if it lasted around 8 seconds or even better: if we could choose its duration with a new option added. 

Of course there are times when several additions to the journal are made during the game, i noticed that specially when i was playing Trials of the Luremaster inside Dragonspear. In this case i'd ask you to be sure/careful to display each of these occurrences for 8 seconds each in sequence. ;)

**Comment by Roberciiik** [comment]
I would say the opposite idea: as you cannot read the journal entry, it could present to you only the entry title and if you click it then the proper journal entry will be opened and displayed to you :)

**Comment by Felipefpl** [comment]
Sorry but i disagree, i dont see why cant i read the summary and then the full content inside the journal, we already do more than that when we are playing the game and reading everything the characters say and then we go to the journal to read even more.

**Comment by Roberciiik** [comment]
It is a matter of taste. I feel this pop-up is redundant. You have an information about journal in the dialog box, and then you can open and see more in the journal itself. Like in the good, old engine.

Instead of increasing the time for reading, it could have a [X] button to just close the entry after you read it... 

Still I am not considering it as a "bug" it is more like UI improvement. Just my opinion.

**Comment by szaumoor** [comment]
I agree this is a taste thing, and it's not a good fit for a bugfix mod. I personally wouldn't want a big wall of text in front of me anyway. Just a short popup that lets me get more details if I want. And that's just me.

**Comment by Felipefpl** [comment]
> It is a matter of taste. I feel this pop-up is redundant. You have an information about journal in the dialog box, and then you can open and see more in the journal itself. Like in the good, old engine.

Indeed, a matter of taste, i'm also favorable to have an option to disable it.

> Instead of increasing the time for reading, it could have a [X] button to just close the entry after you read it...

If this means we'll have an unlimited time to read it i'm all for it.

> Still I am not considering it as a "bug" it is more like UI improvement. Just my opinion.

Indeed a UI improvement, but since there are several things here that arent bugfixes but improvements i say: why not? https://www.gibberlings3.net/subpages/ee-fixpack/docs_bgee/ :P :+1:
